Color/src/Graphics/Color/Space/DIN99.hs

```diff
@@ -0,0 +1,189 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveTraversable #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE Strict #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UnboxedSums #-}
+{-# LANGUAGE UnboxedTuples #-}
+
+module Graphics.Color.Space.DIN99
+  ( pattern DIN99,
+    pattern ColorDIN,
+    DIN99,
+    deltaE,
+  )
+where
+
+import Foreign.Storable
+import GHC.Generics (Generic)
+import Graphics.Color.Illuminant.Wikipedia as W
+import Graphics.Color.Model.Internal
+import Graphics.Color.Space.CIE1976.LAB
+import Graphics.Color.Space.Internal
+
+data DIN99 (i :: k)
+
+-- | Color in DIN99 color space
+newtype instance Color (DIN99 i) e = DIN99 (V3 e)
+
+pattern ColorDIN :: e -> e -> e -> Color (DIN99 i) e
+pattern ColorDIN l' a' b' = DIN99 (V3 l' a' b')
+
+{-# COMPLETE ColorDIN #-}
+
+-- | CIE1976 `DIN99` color space
```

Comment:
I might be wrong, but don't think DIN99 was designed by CIE, so it might be best to remove `CIE1976`  from the haddocks for all instances
```suggestion
-- | `DIN99` color space
```